docs/support/troubleshooting.md

```diff
@@ -1,11 +1,20 @@
 # Troubleshooting
 
-## How to resolve errors of the setup script?
+- [Setup script](#setup-script)
+  - [CUDA-related errors](#cuda-related-errors)
+- [Build errors](#build-errors)
+  - [Insufficient memory](#insufficent-memory)
+  - [Errors when using the latest version of Autoware](#errors-when-using-the-latest-version-of-autoware)
+  - [Errors when using a fixed version of Autoware](#errors-when-using-a-fixed-version-of-autoware)
+- [Docker/rocker issues](#dockerrocker-issues)
+- [Runtime problems](#runtime-problems)
+  - [Map does not display when running the Planning Simulator](#map-does-not-display-when-running-the-planning-simulator)
+
+## Setup script
```

Comment:
Better?

```suggestion
## Setup errors
```

Or
```suggestion
## Setup issues
```

Reply:
I went with `Setup issues` and made the change at the same time that I removed the TOC.